install_config/install/advanced_install.adoc

```diff
@@ -2171,9 +2171,10 @@ ifdef::openshift-origin[]
 endif::[]
 image is a containerized version of the {product-title} installer that runs as a
 link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/managing_containers/running_system_containers[system container]. System containers are stored and run outside of the traditional
-*docker* service. Functionally, using the containerized installer is the same as
-using the traditional RPM-based installer, except it is running in a
-containerized environment instead of directly on the host.
+*docker* service. The installer image provides the same functionality
+as the traditional RPM-based installer, but it runs in a containerized
+environment that provides all of its dependencies rather than being
+installed directly on the host.
```

Comment:
This is the only real advantage -- don't have to install a bunch of RPMs on your host. So I think it's worth mentioning.